test/integration/testpaths/testpath_vmlc.py

```diff
@@ -176,6 +176,19 @@ def setUpClass(cls):
         cls.zone = get_zone(cls.apiclient)
         cls._cleanup = []
 
+        cls.skiptest = False
```

Comment:
There is no need for these checks. Instead the tests which leave the storage pool in disabled state should be looked at. Hold it for now.